Qt.py

```diff
@@ -96,6 +98,36 @@ def _pyside():
     return PySide
 
 
+def set_sip_api_v2(sip_module):
```

Comment:
I don't think this is necessary. A simple try/except inside the `_pyside4` function should be clear enough.